packages/components/toggle-button/src/toggle-button.stories.ts

```diff
@@ -1,66 +1,86 @@
-import { faGear as farGear } from '@fortawesome/pro-regular-svg-icons';
-import { faEyeSlash, faGear as fasGear } from '@fortawesome/pro-solid-svg-icons';
+import { faGear } from '@fortawesome/pro-regular-svg-icons';
+import { faGear as fasGear } from '@fortawesome/pro-solid-svg-icons';
 import { Icon } from '@sl-design-system/icon';
 import '@sl-design-system/icon/register.js';
 import { type Meta, type StoryObj } from '@storybook/web-components';
-import { html } from 'lit';
+import { type TemplateResult, html } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
 import '../register.js';
-import { type ToggleButton, ToggleButtonFill, ToggleButtonSize } from './toggle-button.js';
+import { type ToggleButton } from './toggle-button.js';
 
-interface Props extends Pick<ToggleButton, 'disabled' | 'fill' | 'pressed' | 'size'> {
+type Props = Pick<ToggleButton, 'disabled' | 'fill' | 'pressed' | 'size'> & {
+  icons?: TemplateResult;
   label: string;
-}
+};
 type Story = StoryObj<Props>;
 
-const disabledStates = [false, true];
-const fills: ToggleButtonFill[] = ['ghost', 'outline'];
-const sizes: ToggleButtonSize[] = ['sm', 'md', 'lg'];
-
-Icon.register(fasGear, farGear, faEyeSlash);
+Icon.register(faGear, fasGear);
 
 export default {
   title: 'Components/Toggle button',
-  tags: ['draft'],
+  tags: ['preview'],
   args: {
     disabled: false,
-    fill: 'ghost',
     label: 'Show settings',
-    pressed: false,
-    size: 'md'
+    pressed: false
   },
   argTypes: {
-    size: {
+    fill: {
       control: 'inline-radio',
-      options: sizes
+      options: ['ghost', 'outline']
     },
-    fill: {
+    icons: {
+      table: { disable: true }
+    },
+    size: {
       control: 'inline-radio',
-      options: fills
+      options: ['md', 'lg']
     }
   },
-  render: ({ fill, size, pressed, label, disabled }) => {
-    const onToggle = (event: Event & { target: ToggleButton }): void => {
-      console.log(event.target.pressed);
-    };
-
+  render: ({ disabled, fill, icons, label, pressed, size }) => {
     return html`
```

Comment:
Do we want to show an example of `toggle button` with icon and text next to it?

Reply:
We do not. The users are meant to use that functionality outside of the toggle-group. This was discussed with Danielle.